packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx

```diff
@@ -18,8 +18,9 @@ const AppointmentsMetrics: React.FC<{ serviceUuid: string }> = ({ serviceUuid })
   const { currentAppointmentDate } = useAppointmentDate();
   const formattedStartDate = formatDate(parseDate(currentAppointmentDate), { mode: 'standard', time: false });
 
-  const { appointmentList: arrivedAppointments } = useAppointmentList('Honoured');
-  const { appointmentList: pendingAppointments } = useAppointmentList('Pending');
+  // TODO we will need rework these after we discuss the logic we want to use
+  const { appointmentList: arrivedAppointments } = useAppointmentList('CheckedIn');
+  const { appointmentList: pendingAppointments } = useAppointmentList('Scheduled');
```

Comment:
I fixed to not throw an exception since useAppointmentList needs one of the current appointment statuses, btu we will need to fix what this actually does in the future.